### DISCUSSION TITLE
Is Netbox fully compatible with Ubuntu 24.04 LTS?

### DISCUSSION THREAD

**Discussion by TheGuardianLight** [netbox-community/netbox#18085] [Q&A] [answered]
As a Netbox production user in my company, we wanted to know if there were any compatibility problems with Ubuntu 24.04 LTS.

**Comment by PeterKnotek** [comment]
HI

Answer: Try IT. But answer is ... try install and try netbox.
If Ubuntu has support in the packages that are necessary to run NetBox .... it's OK

**Comment by zmaster7** [comment, marked as answer]
I have been running my test environment on Ubuntu 24.04 LTS since June, and my production since August. I haven't had any issues yet FWIW.

**Reply by TheGuardianLight** [reply]
Thanks for your answer !
